frontend/package.json

```diff
@@ -68,6 +68,7 @@
     "plotly.js": "1.28.x",
     "prop-types": "15.6.x",
     "react": "16.x",
+    "react-ace": "6.1.x",
```

Comment:
We're already using Ace editor elsewhere (edit YAML). I think we should be able to use Ace here without the new dependency. Can you check?

https://github.com/openshift/console/blob/master/frontend/public/components/edit-yaml.jsx

Reply:
Sorry I see you comment on this in the description. I think we should discuss further. How large is the library?

Reply:
from what I see in the node_modules it's 2MB. Just for comparision the `brace` has 8.5MB

Reply:
Don't look at node_modules size. Use `yarn run analyze` to see the size of the built js.

Reply:
react-ace is a thin wrapper around brace:

https://github.com/securingsincity/react-ace/blob/master/package.json#L63:
```javascript
  "dependencies": {
    "brace": "^0.11.0",
   ...
},
```

https://github.com/securingsincity/react-ace/blob/master/src/ace.js#L1:
```javascript
import ace from 'brace'
```

`react-ace` itself is 46.47 KB.  Previously we were async loading `brace` only when needed. Since secret.jsx now imports `brace` (747KB), our main bundle of vendored deps has increased in size by ~40%.  

Are there any interactions here that aren't 90% copy/paste - ie, why do we need a text editor at all?  If you do need one, you should use `AsyncComponent` to load it only when needed since the vast majority of users will not edit a secret.